ResuMe/settings.py

```diff
@@ -130,3 +130,4 @@
 # https://docs.djangoproject.com/en/4.0/ref/settings/#default-auto-field
 
 DEFAULT_AUTO_FIELD = 'django.db.models.BigAutoField'
+LOGIN_REDIRECT_URL = '/'
```

Comment:
Shouldn't it be `LOGIN_REDIRECT_URL ='posts-feed'` as the name of our main URL?

Reply:
> Shouldn't it be `LOGIN_REDIRECT_URL ='posts-feed'` as the name of our main URL?

'/' is the landing page of 'localhost:8000' which is the feed page :)

![image](https://user-images.githubusercontent.com/36789773/162739984-2377f92f-ff4c-4edd-8c3a-ad6b0c0ace8d.png)